src/frontend/src/btc/utils/btc-transactions.utils.ts

```diff
@@ -1,26 +1,42 @@
+import {
+	CONFIRMED_BTC_TRANSACTION_MIN_CONFIRMATIONS,
+	PENDING_BTC_TRANSACTION_MIN_CONFIRMATIONS
+} from '$btc/constants/btc.constants';
 import type { BtcTransactionUi } from '$btc/types/btc';
 import type { BtcAddress } from '$lib/types/address';
 import type { BitcoinOutput, BitcoinTransaction } from '$lib/types/blockchain';
-import { nonNullish } from '@dfinity/utils';
+import { isNullish, nonNullish } from '@dfinity/utils';
 
 export const mapBtcTransaction = ({
 	transaction: { inputs, block_index, out, hash, time },
-	btcAddress
+	btcAddress,
+	latestBitcoinBlockHeight
 }: {
 	transaction: BitcoinTransaction;
 	btcAddress: BtcAddress;
+	latestBitcoinBlockHeight: number;
 }): BtcTransactionUi => {
 	const isTypeSend = inputs.some(({ prev_out }) => prev_out.addr === btcAddress);
-	const blockIndexAvailable = nonNullish(block_index);
 	const output: BitcoinOutput | undefined = out.find(({ addr }) =>
 		isTypeSend ? addr !== btcAddress : addr === btcAddress
 	);
 
+	const confirmations = nonNullish(block_index)
+		? latestBitcoinBlockHeight - block_index
+		: undefined;
+
+	const status =
+		isNullish(confirmations) || confirmations === PENDING_BTC_TRANSACTION_MIN_CONFIRMATIONS
```

Comment:
I think here should be <= right? i know we have 0 for now, but just in case

Reply:
Good point.